packages/core/README.md

```diff
@@ -160,29 +163,12 @@ initializeAgent({
       },
     },
 
-    // Define a meta which does some instant value computation
-    {
-      viewPort: () => ({
-        height: window.innerHeight,
-        width: window.innerWidth,
-      }),
-    },
-
     // Define a meta which caches some values on initialization
     () => {
-      const initialViewport = {
-        height: window.innerHeight,
-        width: window.innerWidth,
-      };
-
       return {
-        initialViewport,
-
-        // You can also define instant metas here
-        viewPort: () => ({
-          height: window.innerHeight,
-          width: window.innerWidth,
-        }),
+        user: {
+          username: getUser().name
```

Comment:
I am not sure about the legal mubmo jumbo, but should we run this through leagal?

Reply:
I don't think so, this is just an example in readme